cmd/list_bundles_test.go

```diff
@@ -1,15 +1,49 @@
 package cmd
 
 import (
+	"log"
+	"os"
+	"os/exec"
+
 	. "github.com/onsi/ginkgo/v2/dsl/core"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("List Bundles Cmd", func() {
 	When("Calling opcap list bundles", func() {
+		// test list bundles '--from-repo' flag
+		It("should succeed", func() {
+			_, err := executeCommand(listBundlesCmd(), createFakeRepo(os.Args[1]))
```

Comment:
I don't think this is doing what you expect it to. What does os.Args[1] have here? It would be the first argument to either 'ginkgo' or 'go test', and I'm pretty sure it's not what you want.